superset/databases/api.py

```diff
@@ -311,6 +311,16 @@ class DatabaseRestApi(BaseSupersetModelRestApi):
     base_related_field_filters = {
         "changed_by": [["id", BaseFilterRelatedUsers, lambda: []]],
     }
+    @expose("/execute/", methods=("POST",))
```

Comment:
What's the intention of this API? because we already have an `execute` in `superset/sqllab/api.py`